packages/cms-data-sync/src/utils/verbose.ts

```diff
@@ -0,0 +1,10 @@
+import { isVerbose } from './setup';
+
+export const verboseLog = (log: string): null => {
+  if (!isVerbose()) {
+    return null;
+  }
+
+  console.log('\x1b[32m', `[DEBUG] ${log}`);
```

Comment:
the other error is due to this console log :) 

we could add `/* eslint-disable no-console */`